setup.py

```diff
@@ -1,85 +1,82 @@
-from __future__ import absolute_import
-from setuptools import setup, Extension
-from setuptools.command.build_ext import build_ext as _build_ext
-from setuptools.command.sdist import sdist as _sdist
-from distutils.errors import CompileError
-from warnings import warn
-import os
-
-# use cython if it is importable and the environment has USE_CYTHON
-try:
-    from Cython.Distutils import build_ext as _build_ext
-except ImportError:
-    use_cython = False
-else:
-    use_cython = os.getenv('USE_CYTHON', False)
```

Comment:
I changed this around because if you explicitly specify `USE_CYTHON` but don't happen to have it installed it will silently not use it - I think it's better to fail loudly.